auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -2,7 +2,29 @@
   <div id="affiliated-entity-section">
     <v-card flat>
       <div class="table-header">
-        <label><strong>My List </strong>({{ entityCount }})</label>
+        <v-row
+        no-gutters
+        class="mb-n8"
+        >
+          <v-col class="pt-3 pl-1">
+            <label><strong>My List </strong>({{ entityCount }})</label>
+          </v-col>
+          <v-col class="column-actions">
+            <v-select
+            dense multiple
+            class="column-selector pb-2 pr-2"
+            background-color="white"
+            label="Columns to Show"
+            v-model="selectedColumns"
+            :items="columns"
+            :menu-props="{ bottom: true, offsetY: true }"
+            @change="updateHeaders"
+            >
```

Comment:
1. do you need "hide-details" (like the old one) to make it less tall?
2. please indent props under the main element tag
3. since you don't need the double-binding on `selectedColumn` (v-model), I suggest changing it to a `value` prop instead (and maybe call it initialColumns or something)